Branch-SDK/Branch-SDK/BNCStrongMatchHelper.m

```diff
@@ -99,24 +104,31 @@ - (void)presentSafariVCWithBranchKey:(NSString *)branchKey {
     if (SFSafariViewControllerClass) {
         id safController = [[SFSafariViewControllerClass alloc] initWithURL:[NSURL URLWithString:urlString]];
         
+        self.secondWindow = [[UIWindow alloc] initWithFrame:[[[[UIApplication sharedApplication] delegate] window] bounds]];
         UIViewController *windowRootController = [[UIViewController alloc] init];
-        
-        self.secondWindow = [[UIWindow alloc] initWithFrame:CGRectZero];
         self.secondWindow.rootViewController = windowRootController;
-        [self.secondWindow makeKeyAndVisible];
+        [self.secondWindow setHidden:NO];
```

Comment:
Is setting this hidden, but not showing it important? Also, does this block UI input?